starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -65,4 +67,21 @@ public void createDaoTest() {
     Assert.assertNotEquals(little, actual.getId());
     Assert.assertFalse(actualInventory.isEmpty());
   }
+  /**
+   * deleteTest checks if the delete method is working as intedned in InventoryDao file.
+   */
+  @Test
+ public void deleteTest() {
+    Inventory inventory = new Inventory();
+    String TEST_ID = "TEST_ID";
```

Comment:
It would make more sense to move this constant up with the other ones. I would also suggest giving it the same modifiers: `private static final`